src/Components/Card.js

```diff
@@ -45,6 +45,8 @@ export default class Card {
 
   deleteCard() {
     const cardId = this._data._id;
+    this._element.remove();
+    this._element = null;
```

Comment:
Please, see my advice again here https://github.com/travis-sutton/se_project_aroundtheus/pull/1#discussion_r1466367480 

You need to **delete** cards only in block `then` after a **successful** response from the server so that if there is an error the card would stay in the `DOM`.


Or you can do it like this here. Add `then` with the deleting

```js

  deleteCard() {
    const cardId = this._data._id;
    return this._api.deleteCard(cardId).then(() => {
      this._element.remove();
      this._element = null;
    });
  }
```